.azure-pipelines/bazel.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
+- name: ciTarget
+  displayName: "CI target"
+  type: string
+  default: build 
+
+steps:
+- bash: |
+    echo "disk space at beginning of build:"
+    df -h
+  displayName: "Check disk space at beginning"
+
+- script: ci/run_envoy_docker.sh 'ci/do_ci.sh ${{ parameters.ciTarget }}'
```

Comment:
Do we know for a fact that we need the `ci/run_envoy_docker.sh`? It would be easier if we could make it work without that dependency, since it will require regular updates / maintenance.

Maybe we could first try without and add it here if we cannot make the CI work without?

Reply:
Yes, got it. Sorry I just copied from Nighthawk's AZP configuration. But if we want to run without `ci/run_envoy_docker.sh`, we also need to find a way to make CI workflow run in a correct Envoy docker container, like https://github.com/envoyproxy/envoy-perf/blob/main/.circleci/config.yml#L3. 
At first, we can draft a simple script to implement it. And if we need more features in `ci/run_envoy_docker.sh` really, we can use it again. Does it make sense?

Reply:
Thinking more, it might be simplest if we just copy and use the same script. We will need the container for the tooling that it already contains. And if we use the same script in all three repos, it will be easier to debug and to understand the CI setup.

Going to resolve this conversation. We can continue discussing how to document the maintenance procedure in the other comment thread.